docs/utilization.md

```diff
@@ -0,0 +1,116 @@
+# Utilization reports in Tock
+
+[:arrow_left: Back to Tock Documentation](../docs)
+
+
+Tock utilization reports display progress towards billing targets
+for each user, business unit, and the overall organization.
+
+Each user has a billing target which is the percentage of worked hours that are expected to be tocked to billable projects.
```

Comment:
Can't remember: in the admin, do we express the target as a number of hours, or as a percentage?

Reply:
Percentage